launch-game.cmd

```diff
@@ -5,18 +5,18 @@ title OpenRA
 FOR /F "tokens=1,2 delims==" %%A IN (mod.config) DO (set %%A=%%B)
 if exist user.config (FOR /F "tokens=1,2 delims==" %%A IN (user.config) DO (set %%A=%%B))
 set TEMPLATE_LAUNCHER=%0
-set MOD_SEARCH_PATHS=%~dp0mods,./mods
+set MOD_SEARCH_PATHS=%~dp0mods,../mods

```

Comment:
Per the OpenRA SDK this should stay as it was.

Reply:
I copied glyphs.png from RA and it seems to work "ok" in that I see the flags and such that I expect. Copying chrome.png I saw pieces of the ui bars instead of flags.

Do you know where glyphs.png should come from normally?

Reply:
https://github.com/OpenRA/OpenRA/commit/128265027423842351cc97ab4d36ebfbd9d688da
This is the commit that introduced them in the engine.
I believe they are modified and split up versions of the original.

Interesting though, I don't see any artifacts using them on Windows.
Might be also why it didn't cause issues on linux.